sdk/digitaltwins/Azure.DigitalTwins.Core/tests/NewtonsoftObjectSerializerTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Core.Serialization;
+using NUnit.Framework;
+
+namespace Azure.DigitalTwins.Core.Tests
+{
+    /// <summary>
+    /// Tests for custom ObjectSerializer.
```

Comment:
Maybe say why this is important

Reply:
Specifically, we have a work around in the query code which requires use of a System.Text.Json serializer because it works on a JsonElement. When the user initializes with a non-default serializer, we must not use that to do this work.